orca-core/src/main/groovy/com/netflix/spinnaker/orca/config/OrcaConfiguration.groovy

```diff
@@ -83,7 +81,8 @@ class OrcaConfiguration {
     Duration.of(3, MINUTES)
   }
 
-  @Bean Scheduler scheduler() {
+  @Bean(destroyMethod = "")
```

Comment:
newer RxJava scheduler adds a `close` method which Spring was autodiscovering and ruining multiple test case executions